apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -54,7 +54,10 @@ defmodule SiteWeb.ScheduleController.TimetableController do
     |> assign(:trip_schedules, trip_schedules)
     |> assign(:vehicle_schedules, vehicle_schedules)
     |> assign(:prior_stops, prior_stops)
-    |> assign(:trip_messages, trip_messages(conn.assigns.route, conn.assigns.direction_id))
+    |> assign(
+      :trip_messages,
+      trip_messages(conn.assigns.route, conn.assigns.direction_id, conn.assigns.date)
```

Comment:
looks like this update may have broken a test at `test/site_web/controllers/schedule_controller_test.exs:100`

Reply:
I fixed this by excluding the franklin/fairmount from these tests since they are date dependent (and would start failing on Oct 21 or would have to be hardcoded to a date, both seem meh) and I already covered the date changing case in another test.